### ISSUE TITLE
Upgrade from 1.8.1-RC.1 to 1.91 results in inaccessible webinterface (E500)

### ISSUE THREAD

**Issue by jerryhopper** [FusionAuth/fusionauth-issues#306]
### Description 
After update the web administration is inaccessible.

### Steps to reproduce
1. Upgrade from 1.8.1-RC.1 to 1.91
2. Goto the web-login of your fusionauth instance.
3. See the Error.

### Expected behavior
A login-screen should occur.

### Additional context
Felix Langfelt posted a errorlog on SLACK : 
`Sep 24, 2019 8:38:35.620 AM ERROR io.fusionauth.app.primeframework.error.ExceptionExceptionHandler - An unhandled exception was thrown
java.lang.NullPointerException: null
    at io.fusionauth.app.action.BaseThemedAction.resolveTenantAndTheme(BaseThemedAction.java:93)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke(Method.java:498)
    at org.primeframework.mvc.util.ReflectionUtils.invokeAll(ReflectionUtils.java:458)
    at org.primeframework.mvc.parameter.DefaultPostParameterHandler.handle(DefaultPostParameterHandler.java:32)
    at org.primeframework.mvc.parameter.DefaultPostParameterWorkflow.invokeMethods(DefaultPostParameterWorkflow.java:54)
    at org.primeframework.mvc.parameter.DefaultPostParameterWorkflow.perform(DefaultPostParameterWorkflow.java:48)
    at org.primeframework.mvc.workflow.SubWorkflowChain.continueWorkflow(SubWorkflowChain.java:43)
    at org.primeframework.mvc.content.DefaultContentWorkflow.perform(DefaultContentWorkflow.java:52)
    at org.primeframework.mvc.workflow.SubWorkflowChain.continueWorkflow(SubWorkflowChain.java:43)
    at org.primeframework.mvc.parameter.DefaultParameterWorkflow.perform(DefaultParameterWorkflow.java:57)
    at org.primeframework.mvc.workflow.SubWorkflowChain.continueWorkflow(SubWorkflowChain.java:43)
    at org.primeframework.mvc.parameter.DefaultURIParameterWorkflow.perform(DefaultURIParameterWorkflow.java:102)
    at org.primeframework.mvc.workflow.SubWorkflowChain.continueWorkflow(SubWorkflowChain.java:43)
    at org.primeframework.mvc.scope.DefaultScopeRetrievalWorkflow.perform(DefaultScopeRetrievalWorkflow.java:58)
    at org.primeframework.mvc.workflow.SubWorkflowChain.continueWorkflow(SubWorkflowChain.java:43)
    at org.primeframework.mvc.message.DefaultMessageWorkflow.perform(DefaultMessageWorkflow.java:45)
    at org.primeframework.mvc.workflow.SubWorkflowChain.continueWorkflow(SubWorkflowChain.java:43)
    at io.fusionauth.app.primeframework.FrontEndTenantWorkflow.perform(FrontEndTenantWorkflow.java:133)
    at org.primeframework.mvc.workflow.SubWorkflowChain.continueWorkflow(SubWorkflowChain.java:43)
    at org.primeframework.mvc.action.DefaultActionMappingWorkflow.perform(DefaultActionMappingWorkflow.java:126)
    at org.primeframework.mvc.workflow.SubWorkflowChain.continueWorkflow(SubWorkflowChain.java:43)
    at org.primeframework.mvc.workflow.StaticResourceWorkflow.perform(StaticResourceWorkflow.java:97)
    at org.primeframework.mvc.workflow.SubWorkflowChain.continueWorkflow(SubWorkflowChain.java:43)
    at org.primeframework.mvc.parameter.RequestBodyWorkflow.perform(RequestBodyWorkflow.java:89)
    at org.primeframework.mvc.workflow.SubWorkflowChain.continueWorkflow(SubWorkflowChain.java:43)
    at org.primeframework.mvc.security.DefaultSavedRequestWorkflow.perform(DefaultSavedRequestWorkflow.java:57)
    at org.primeframework.mvc.workflow.SubWorkflowChain.continueWorkflow(SubWorkflowChain.java:43)
    at io.fusionauth.app.primeframework.CORSFilter.handleNonCORS(CORSFilter.java:289)
    at io.fusionauth.app.primeframework.CORSFilter.doFilter(CORSFilter.java:250)
    at io.fusionauth.app.primeframework.CORSRequestWorkflow.perform(CORSRequestWorkflow.java:48)
    at org.primeframework.mvc.workflow.SubWorkflowChain.continueWorkflow(SubWorkflowChain.java:43)
    at io.fusionauth.app.primeframework.FusionAuthMVCWorkflow.perform(FusionAuthMVCWorkflow.java:88)
    at org.primeframework.mvc.workflow.DefaultWorkflowChain.continueWorkflow(DefaultWorkflowChain.java:44)
    at org.primeframework.mvc.servlet.FilterWorkflowChain.continueWorkflow(FilterWorkflowChain.java:50)
    at org.primeframework.mvc.servlet.PrimeFilter.doFilter(PrimeFilter.java:84)
    at com.inversoft.maintenance.servlet.MaintenanceModePrimeFilter.doFilter(MaintenanceModePrimeFilter.java:59)
    at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193)
    at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
    at com.inversoft.servlet.UTF8Filter.doFilter(UTF8Filter.java:27)
    at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193)
    at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
    at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:199)
    at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:96)
    at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:493)
    at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:137)
    at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:81)
    at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:87)
    at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:343)
    at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:798)
    at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:66)
    at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:808)
    at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1498)
    at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
    at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
    at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
    at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
    at java.lang.Thread.run(Thread.java:748)`

**Comment by jerryhopper** [comment]
According to Felix Langfeldt 2:57 PM
> I installed a new server and that worked fine. Looks like the default tenant name changed from default to Default. No idea if this makes a big difference.

**Comment by robotdan** [comment]
Thanks for reporting, looking at it.

**Comment by jerryhopper** [comment]
My log from a docker instance : 
`fusionauth_1  | Sep 24, 2019 1:09:49.526 PM ERROR io.fusionauth.app.primeframework.error.ExceptionExceptionHandler - An unhandled exception was thrown
fusionauth_1  | java.lang.NullPointerException: null
fusionauth_1  |         at io.fusionauth.app.action.BaseThemedAction.resolveTenantAndTheme(BaseThemedAction.java:93)
fusionauth_1  |         at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
fusionauth_1  |         at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
fusionauth_1  |         at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
fusionauth_1  |         at java.lang.reflect.Method.invoke(Method.java:498)
fusionauth_1  |         at org.primeframework.mvc.util.ReflectionUtils.invokeAll(ReflectionUtils.java:458)
fusionauth_1  |         at org.primeframework.mvc.parameter.DefaultPostParameterHandler.handle(DefaultPostParameterHandler.java:32)
fusionauth_1  |         at org.primeframework.mvc.parameter.DefaultPostParameterWorkflow.invokeMethods(DefaultPostParameterWorkflow.java:54)
fusionauth_1  |         at org.primeframework.mvc.parameter.DefaultPostParameterWorkflow.perform(DefaultPostParameterWorkflow.java:48)
fusionauth_1  |         at org.primeframework.mvc.workflow.SubWorkflowChain.continueWorkflow(SubWorkflowChain.java:43)
fusionauth_1  |         at org.primeframework.mvc.content.DefaultContentWorkflow.perform(DefaultContentWorkflow.java:52)
fusionauth_1  |         at org.primeframework.mvc.workflow.SubWorkflowChain.continueWorkflow(SubWorkflowChain.java:43)
fusionauth_1  |         at org.primeframework.mvc.parameter.DefaultParameterWorkflow.perform(DefaultParameterWorkflow.java:57)
fusionauth_1  |         at org.primeframework.mvc.workflow.SubWorkflowChain.continueWorkflow(SubWorkflowChain.java:43)
fusionauth_1  |         at org.primeframework.mvc.parameter.DefaultURIParameterWorkflow.perform(DefaultURIParameterWorkflow.java:102)
fusionauth_1  |         at org.primeframework.mvc.workflow.SubWorkflowChain.continueWorkflow(SubWorkflowChain.java:43)
fusionauth_1  |         at org.primeframework.mvc.scope.DefaultScopeRetrievalWorkflow.perform(DefaultScopeRetrievalWorkflow.java:58)
fusionauth_1  |         at org.primeframework.mvc.workflow.SubWorkflowChain.continueWorkflow(SubWorkflowChain.java:43)
fusionauth_1  |         at org.primeframework.mvc.message.DefaultMessageWorkflow.perform(DefaultMessageWorkflow.java:45)
fusionauth_1  |         at org.primeframework.mvc.workflow.SubWorkflowChain.continueWorkflow(SubWorkflowChain.java:43)
fusionauth_1  |         at io.fusionauth.app.primeframework.FrontEndTenantWorkflow.perform(FrontEndTenantWorkflow.java:133)
fusionauth_1  |         at org.primeframework.mvc.workflow.SubWorkflowChain.continueWorkflow(SubWorkflowChain.java:43)
fusionauth_1  |         at org.primeframework.mvc.action.DefaultActionMappingWorkflow.perform(DefaultActionMappingWorkflow.java:126)
fusionauth_1  |         at org.primeframework.mvc.workflow.SubWorkflowChain.continueWorkflow(SubWorkflowChain.java:43)
fusionauth_1  |         at org.primeframework.mvc.workflow.StaticResourceWorkflow.perform(StaticResourceWorkflow.java:97)
fusionauth_1  |         at org.primeframework.mvc.workflow.SubWorkflowChain.continueWorkflow(SubWorkflowChain.java:43)
fusionauth_1  |         at org.primeframework.mvc.parameter.RequestBodyWorkflow.perform(RequestBodyWorkflow.java:89)
fusionauth_1  |         at org.primeframework.mvc.workflow.SubWorkflowChain.continueWorkflow(SubWorkflowChain.java:43)
fusionauth_1  |         at org.primeframework.mvc.security.DefaultSavedRequestWorkflow.perform(DefaultSavedRequestWorkflow.java:57)
fusionauth_1  |         at org.primeframework.mvc.workflow.SubWorkflowChain.continueWorkflow(SubWorkflowChain.java:43)
fusionauth_1  |         at io.fusionauth.app.primeframework.CORSFilter.handleNonCORS(CORSFilter.java:289)
fusionauth_1  |         at io.fusionauth.app.primeframework.CORSFilter.doFilter(CORSFilter.java:250)
fusionauth_1  |         at io.fusionauth.app.primeframework.CORSRequestWorkflow.perform(CORSRequestWorkflow.java:48)
fusionauth_1  |         at org.primeframework.mvc.workflow.SubWorkflowChain.continueWorkflow(SubWorkflowChain.java:43)
fusionauth_1  |         at io.fusionauth.app.primeframework.FusionAuthMVCWorkflow.perform(FusionAuthMVCWorkflow.java:88)
fusionauth_1  |         at org.primeframework.mvc.workflow.DefaultWorkflowChain.continueWorkflow(DefaultWorkflowChain.java:44)
fusionauth_1  |         at org.primeframework.mvc.servlet.FilterWorkflowChain.continueWorkflow(FilterWorkflowChain.java:50)
fusionauth_1  |         at org.primeframework.mvc.servlet.PrimeFilter.doFilter(PrimeFilter.java:84)
fusionauth_1  |         at com.inversoft.maintenance.servlet.MaintenanceModePrimeFilter.doFilter(MaintenanceModePrimeFilter.java:59)
fusionauth_1  |         at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193)
fusionauth_1  |         at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
fusionauth_1  |         at com.inversoft.servlet.UTF8Filter.doFilter(UTF8Filter.java:27)
fusionauth_1  |         at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193)
fusionauth_1  |         at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
fusionauth_1  |         at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:199)
fusionauth_1  |         at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:96)
fusionauth_1  |         at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:493)
fusionauth_1  |         at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:137)
fusionauth_1  |         at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:81)
fusionauth_1  |         at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:87)
fusionauth_1  |         at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:343)
fusionauth_1  |         at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:798)
fusionauth_1  |         at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:66)
fusionauth_1  |         at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:808)
fusionauth_1  |         at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1498)
fusionauth_1  |         at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
fusionauth_1  |         at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
fusionauth_1  |         at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
fusionauth_1  |         at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
fusionauth_1  |         at java.lang.Thread.run(Thread.java:748)
fusionauth_1  | Sep 24, 2019 1:09:49.526 PM ERROR io.fusionauth.app.primeframework.error.ExceptionExceptionHandler - An unhandled exception was thrown
fusionauth_1  | java.lang.NullPointerException: null
fusionauth_1  |         at io.fusionauth.app.action.BaseThemedAction.resolveTenantAndTheme(BaseThemedAction.java:93)
fusionauth_1  |         at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
fusionauth_1  |         at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
fusionauth_1  |         at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
fusionauth_1  |         at java.lang.reflect.Method.invoke(Method.java:498)
fusionauth_1  |         at org.primeframework.mvc.util.ReflectionUtils.invokeAll(ReflectionUtils.java:458)
fusionauth_1  |         at org.primeframework.mvc.parameter.DefaultPostParameterHandler.handle(DefaultPostParameterHandler.java:32)
fusionauth_1  |         at org.primeframework.mvc.parameter.DefaultPostParameterWorkflow.invokeMethods(DefaultPostParameterWorkflow.java:54)
fusionauth_1  |         at org.primeframework.mvc.parameter.DefaultPostParameterWorkflow.perform(DefaultPostParameterWorkflow.java:48)
fusionauth_1  |         at org.primeframework.mvc.workflow.SubWorkflowChain.continueWorkflow(SubWorkflowChain.java:43)
fusionauth_1  |         at org.primeframework.mvc.content.DefaultContentWorkflow.perform(DefaultContentWorkflow.java:52)
fusionauth_1  |         at org.primeframework.mvc.workflow.SubWorkflowChain.continueWorkflow(SubWorkflowChain.java:43)
fusionauth_1  |         at org.primeframework.mvc.parameter.DefaultParameterWorkflow.perform(DefaultParameterWorkflow.java:57)
fusionauth_1  |         at org.primeframework.mvc.workflow.SubWorkflowChain.continueWorkflow(SubWorkflowChain.java:43)
fusionauth_1  |         at org.primeframework.mvc.parameter.DefaultURIParameterWorkflow.perform(DefaultURIParameterWorkflow.java:102)
fusionauth_1  |         at org.primeframework.mvc.workflow.SubWorkflowChain.continueWorkflow(SubWorkflowChain.java:43)
fusionauth_1  |         at org.primeframework.mvc.scope.DefaultScopeRetrievalWorkflow.perform(DefaultScopeRetrievalWorkflow.java:58)
fusionauth_1  |         at org.primeframework.mvc.workflow.SubWorkflowChain.continueWorkflow(SubWorkflowChain.java:43)
fusionauth_1  |         at org.primeframework.mvc.message.DefaultMessageWorkflow.perform(DefaultMessageWorkflow.java:45)
fusionauth_1  |         at org.primeframework.mvc.workflow.SubWorkflowChain.continueWorkflow(SubWorkflowChain.java:43)
fusionauth_1  |         at io.fusionauth.app.primeframework.FrontEndTenantWorkflow.perform(FrontEndTenantWorkflow.java:133)
fusionauth_1  |         at org.primeframework.mvc.workflow.SubWorkflowChain.continueWorkflow(SubWorkflowChain.java:43)
fusionauth_1  |         at org.primeframework.mvc.action.DefaultActionMappingWorkflow.perform(DefaultActionMappingWorkflow.java:126)
fusionauth_1  |         at org.primeframework.mvc.workflow.SubWorkflowChain.continueWorkflow(SubWorkflowChain.java:43)
fusionauth_1  |         at org.primeframework.mvc.workflow.StaticResourceWorkflow.perform(StaticResourceWorkflow.java:97)
fusionauth_1  |         at org.primeframework.mvc.workflow.SubWorkflowChain.continueWorkflow(SubWorkflowChain.java:43)
fusionauth_1  |         at org.primeframework.mvc.parameter.RequestBodyWorkflow.perform(RequestBodyWorkflow.java:89)
fusionauth_1  |         at org.primeframework.mvc.workflow.SubWorkflowChain.continueWorkflow(SubWorkflowChain.java:43)
fusionauth_1  |         at org.primeframework.mvc.security.DefaultSavedRequestWorkflow.perform(DefaultSavedRequestWorkflow.java:57)
fusionauth_1  |         at org.primeframework.mvc.workflow.SubWorkflowChain.continueWorkflow(SubWorkflowChain.java:43)
fusionauth_1  |         at io.fusionauth.app.primeframework.CORSFilter.handleNonCORS(CORSFilter.java:289)
fusionauth_1  |         at io.fusionauth.app.primeframework.CORSFilter.doFilter(CORSFilter.java:250)
fusionauth_1  |         at io.fusionauth.app.primeframework.CORSRequestWorkflow.perform(CORSRequestWorkflow.java:48)
fusionauth_1  |         at org.primeframework.mvc.workflow.SubWorkflowChain.continueWorkflow(SubWorkflowChain.java:43)
fusionauth_1  |         at io.fusionauth.app.primeframework.FusionAuthMVCWorkflow.perform(FusionAuthMVCWorkflow.java:88)
fusionauth_1  |         at org.primeframework.mvc.workflow.DefaultWorkflowChain.continueWorkflow(DefaultWorkflowChain.java:44)
fusionauth_1  |         at org.primeframework.mvc.servlet.FilterWorkflowChain.continueWorkflow(FilterWorkflowChain.java:50)
fusionauth_1  |         at org.primeframework.mvc.servlet.PrimeFilter.doFilter(PrimeFilter.java:84)
fusionauth_1  |         at com.inversoft.maintenance.servlet.MaintenanceModePrimeFilter.doFilter(MaintenanceModePrimeFilter.java:59)
fusionauth_1  |         at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193)
fusionauth_1  |         at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
fusionauth_1  |         at com.inversoft.servlet.UTF8Filter.doFilter(UTF8Filter.java:27)
fusionauth_1  |         at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:193)
fusionauth_1  |         at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:166)
fusionauth_1  |         at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:199)
fusionauth_1  |         at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:96)
fusionauth_1  |         at org.apache.catalina.authenticator.AuthenticatorBase.invoke(AuthenticatorBase.java:493)
fusionauth_1  |         at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:137)
fusionauth_1  |         at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:81)
fusionauth_1  |         at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:87)
fusionauth_1  |         at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:343)
fusionauth_1  |         at org.apache.coyote.http11.Http11Processor.service(Http11Processor.java:798)
fusionauth_1  |         at org.apache.coyote.AbstractProcessorLight.process(AbstractProcessorLight.java:66)
fusionauth_1  |         at org.apache.coyote.AbstractProtocol$ConnectionHandler.process(AbstractProtocol.java:808)
fusionauth_1  |         at org.apache.tomcat.util.net.NioEndpoint$SocketProcessor.doRun(NioEndpoint.java:1498)
fusionauth_1  |         at org.apache.tomcat.util.net.SocketProcessorBase.run(SocketProcessorBase.java:49)
fusionauth_1  |         at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
fusionauth_1  |         at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
fusionauth_1  |         at org.apache.tomcat.util.threads.TaskThread$WrappingRunnable.run(TaskThread.java:61)
fusionauth_1  |         at java.lang.Thread.run(Thread.java:748)
`

**Comment by robotdan** [comment]
@jerryhopper @flangfeldt 

This is really strange, not sure how this is possible. Can you tell me anything about your environment prior to upgrade? 

1. How many themes did you have defined? 
2. How many tenants do you have? 
3. Is a theme, other than the default assigned to each tenant? 


Thanks!

**Comment by jerryhopper** [comment]
Hmm, 
its a default docker instance.
No themes.
Maybe 1 (unused) tenant for test (cant remember.)

Here is my full startuplog. https://jerryhopper.com/FA_logs.txt

Have you checked the name of the default tenant ?  (default/Default)

**Comment by robotdan** [comment]
Thanks @jerryhopper that log is super helpful. 

The root cause is this

```
[36mfusionauth_1  |[0m Sep 24, 2019 12:58:52.586 PM ERROR com.inversoft.scheduler.LogAndRetainFailureHandler - The scheduled service [class io.fusionauth.api.service.cache.ThemeCacheLoader] failed but will be re-run.
[36mfusionauth_1  |[0m java.lang.IllegalStateException: Invalid message value for theme
[36mfusionauth_1  |[0m 	at io.fusionauth.api.domain.CachedTheme.loadProperties(CachedTheme.java:40)
[36mfusionauth_1  |[0m 	at io.fusionauth.api.domain.CachedTheme.<init>(CachedTheme.java:24)
[36mfusionauth_1  |[0m 	at java.util.stream.Collectors.lambda$toMap$58(Collectors.java:1321)
[36mfusionauth_1  |[0m 	at java.util.stream.ReduceOps$3ReducingSink.accept(ReduceOps.java:169)
[36mfusionauth_1  |[0m 	at java.util.ArrayList$ArrayListSpliterator.forEachRemaining(ArrayList.java:1382)
[36mfusionauth_1  |[0m 	at java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:481)
[36mfusionauth_1  |[0m 	at java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:471)
[36mfusionauth_1  |[0m 	at java.util.stream.ReduceOps$ReduceOp.evaluateSequential(ReduceOps.java:708)
[36mfusionauth_1  |[0m 	at java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
[36mfusionauth_1  |[0m 	at java.util.stream.ReferencePipeline.collect(ReferencePipeline.java:499)
[36mfusionauth_1  |[0m 	at io.fusionauth.api.service.cache.ThemeCacheLoader.load(ThemeCacheLoader.java:34)
[36mfusionauth_1  |[0m 	at io.fusionauth.api.service.cache.ThemeCacheLoader.run(ThemeCacheLoader.java:40
```

Not sure why this is failing, but it must not like something in the resource bundle.

**Comment by flangfeldt** [comment]
I had 1 tenant and 2 themes. But I have created and deleted tenants prior to the upgrade.
The tenant was assigned to the 2nd theme (non-standard).

**Comment by robotdan** [comment]
> Have you checked the name of the default tenant ? (default/Default)

This should be ok, the name is mutable.

**Comment by robotdan** [comment]
I believe I have this nailed down, fix forthcoming.

**Comment by robotdan** [comment]
Duplicated issue on SO.
https://stackoverflow.com/questions/58083668/internal-server-error-after-fusion-auth-upgrade-to-v-1-9-1

**Comment by robotdan** [comment]
Available in 1.9.2

**Comment by simo97** [comment]
Hi this problem occurs in a case where I am migrating the database from one server to another one. I am on the version 1.36.4. And for some reason the API as well did responded.
```
{
	"generalErrors": [
		{
			"code": "[Exception]",
			"message": "FusionAuth encountered an unexpected error. Please review the troubleshooting guide found in the documentation for assistance and the available support channels."
		}
	]
}
```

I had to connect to the SQL table and set the data field to `{}` for it to work normally, but the application is showing `Requires upgrade` aside of the theme name.
